SimonsAlgorithm/BooleanMatrix.cs

```diff
@@ -47,7 +47,7 @@ public BooleanVector(IEnumerable<long> array)
                 throw new ArgumentException($"Vectors have different sizes [{vector1.Count} != {vector2.Count}]");
             }
 
-            return new BooleanVector(vector1.Zip(vector2, (val1, val2) => val1 ^ val2));
```

Comment:
This is not a Q# file. Therefore, you do not need to update this `Zip` function to `Zipped`